sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -1127,7 +1129,8 @@ private case class CosmosChangeFeedConfig
   startFrom: ChangeFeedStartFromMode,
   startFromPointInTime: Option[Instant],
   maxItemCountPerTrigger: Option[Long],
-  batchCheckpointLocation: Option[String]
+  batchCheckpointLocation: Option[String],
+  ignoreOffsetWhenInvalid: Boolean
```

Comment:
I wonder what is the difference between Option[String], Option[Long] and just using Boolean for this property? For others as well, we could have relied on null values, no?

Reply:
Null is an absolute no-go in scala (raises warnings everywhere)

Reply:
Cannot be avoided when calling java apis always - but in scala Option is the way to go